cmd/mdatagen/templates/metrics_test.go.tmpl

```diff
@@ -126,10 +127,13 @@ func TestMetricsBuilder(t *testing.T) {
 
 			assert.Equal(t, 1, rm.ScopeMetrics().Len())
 			ms := rm.ScopeMetrics().At(0).Metrics()
-			if test.metricsSet == testMetricsSetDefault {
+			if test.configSet == testSetDefault {
 				assert.Equal(t, defaultMetricsCount, ms.Len())
 			}
-			if test.metricsSet == testMetricsSetAll {
+			if test.configSet == testSetNone {
+				assert.Equal(t, 0, ms.Len())
```

Comment:
This branch is never executed. We have the same condition above with the return statement